d3a_interface/constants_limits.py

```diff
@@ -64,6 +64,10 @@ class SettlementMarketSettings:
         ENABLE_SETTLEMENT_MARKETS = False
         RELATIVE_STD_FROM_FORECAST_FLOAT = 10.0
 
+    class FutureMarketSettings:
+        # Duration between clearing in future markets
+        FUTURE_MARKET_CLEARING_INTERVAL = 15  # in minutes
```

Comment:
As commented in the other PR, I would prefer FUTURE_MARKET_CLEARING_INTERVAL_M or FUTURE_MARKET_CLEARING_INTERVAL_MINUTES in order to clarify the unit of the constant.

Reply:
Whats concluded here?

Reply:
Nevermind, just saw that its resolved via https://github.com/gridsingularity/gsy-framework/pull/285/commits/18e7081a82a7650e8cd4ef63938cf5a65205cf08

Reply:
@faizan2590 I went for `FUTURE_MARKET_CLEARING_INTERVAL_MINUTES`